ckanext/dbquery/plugin.py

```diff
@@ -36,3 +37,7 @@ def get_auth_functions(self):
         return {
             "query_database": auth.query_database,
         }
+
+    # IConfigurable
+    def configure(self, config):
+        pass
```

Comment:
The `IConfigurable` Interface only have a single function `configure` 
We are not doing anything here.
This is probably not required